infra/examples/aws-msft-365/variables.tf

```diff
@@ -49,3 +49,8 @@ variable "certificate_subject" {
   type        = string
   description = "value for 'subject' passed to openssl when generation certificate (eg '/C=US/ST=New York/L=New York/CN=www.worklytics.co')"
 }
+
+variable "psoxy_basedir" {
+  type        = string
+  description = "the path where your psoxy repo resides"
```

Comment:
add default value (`../../..`)

Reply:
Did not add it on purpose, so it has to be specified, but OK.